rotkehlchen/chain/manager.py

```diff
@@ -1209,10 +1216,9 @@ def add_ethereum_lp_to_balance(
                 else:
                     balances[loc_key][EvmToken(ethaddress_to_identifier(lp.address))] = lp.user_balance  # noqa: E501
 
-    def _add_protocol_balances(self) -> None:
+    def _add_protocol_balances(self, balances: DefaultDict[ChecksumEvmAddress, BalanceSheet]) -> None:  # noqa: E501
```

Comment:
no no this is ethereum specific -- least for now. You should not rename it to `balances`